### PR TITLE
Use Github Actions for Windows CMake build

### PR DESCRIPTION
**Pull request by o01eg**

### CODE REVIEW

**Comment by geoffthemedio** [comment]
no comment on most, and no objections to adding an include you need

File: .github/workflows/build-windows.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: "Windows build"
+
+on:
+  push:
```
Comment (Cjkjvfnby) [inline .github/workflows/build-windows.yml:4]: Should we really build it on each push?

As I recall Geoff uses this reap to push his development branches and probably all other forks will be affected.

File: .github/workflows/build-windows.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    name: Build FreeOrion on Windows
+    runs-on: windows-2019
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+      - name: Download SDK
+        id: download-sdk
+        uses: suisei-cn/actions-download-file@v1
+        with:
+          url: https://github.com/freeorion/freeorion-sdk/releases/download/v11/FreeOrionSDK_11_MSVC-v141-Win32.zip
```
Comment (Cjkjvfnby) [inline .github/workflows/build-windows.yml:32]: Should we cache it?  https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows  ()

Comment (o01eg) [reply .github/workflows/build-windows.yml:32]: Hard to say if Github Cache quicker than Github Releases file. It better to have something like ccache for MSVS.

File: .github/workflows/build-windows.yml (hunk 1 above)
Comment (o01eg) [reply .github/workflows/build-windows.yml:4]: Current Travis, Appveyor and Github CI build project on each push.

Comment (geoffthemedio) [reply .github/workflows/build-windows.yml:4]: Is there a reason not to?

**Review by Vezzra — COMMENTED** [review]
Can't really provide any meaningful input... as far as I can tell, looks fine...

**Comment by geoffthemedio** [comment]
I suppose the caching issue can be discussed separately...

**Pull request merged**